src/Tile/README.md

```diff
@@ -0,0 +1,19 @@
+# ⚠️ This Section is a WIP ⚠️
+
+
+# Tile
+> [UIKit documentation](https://getuikit.com/docs/Tile)
```

Comment:
Documentation is case sensitive, leaving this link broken. See: [https://getuikit.com/docs/tile](https://getuikit.com/docs/tile)